client/src/js/components/bhDebtorGroupSelect.js

```diff
@@ -0,0 +1,51 @@
+angular.module('bhima.components')
+  .component('bhDebtorGroupSelect', {
+    templateUrl : 'modules/templates/bhDebtorGroupSelect.tmpl.html',
+    controller  : DebtorGroupSelectController,
+    // transclude  : true,
```

Comment:
I think it is necessary to allow this component to `transclude` to allow you to use the `bh-clear` component inside (in search modals for example).